src/Symfony/Bundle/FrameworkBundle/Controller/AbstractController.php

```diff
@@ -226,9 +226,13 @@ protected function denyAccessUnlessGranted(mixed $attribute, mixed $subject = nu
      * Returns a rendered view.
      *
      * Forms found in parameters are auto-cast to form views.
+     *
+     * @param string|null $block
      */
-    protected function renderView(string $view, array $parameters = []): string
+    protected function renderView(string $view, array $parameters = []/* , string $block = null */): string
```

Comment:
Sorry I dont see why adding this new argument like it, will be better than adding a new method `renderViewBlock()`, just like you did for `renderBlock()`, can you please elaborate a bit more about it?

Reply:
IMHO, they've the same reason as `renderBlock`:
* no deprecation warning for subclasses
* imho better argument order `renderViewBlock('foo.html.twig', 'block_name', [...])` aligned with other render. methods

Reply:
That's because I have some reluctance to adding more and more methods that are slight variations of each others in AbstractController, which is already a long collection of helpers.

`$this->render('foo.html.twig', block: 'bar', $context)` would read equally well to me compared to `renderBlock()` but many have expressed their preference so I abide by them :) 

For `renderBlockView`, my reluctance is still there, because yet another method, while this method won't be much used (we didn't miss it for now). Also `$this->renderView('foo.html.twig', block: 'bar', $context)` reads nicely so keep AbstractController shorter in terms of number of methods has my preference.

Reply:
I understand your concern as a maintainer, and it makes sense to me. However, from the DX perspective, it will create some kind of inconsistency in how `renderBlock` and `renderView` are used (while your goal is rendering blocks). I think adding one more method doesn't hurt too much if it results in a more intuitive API

Reply:
Thus, our BC policy current states that parameter names are not covered by BC except for attributes, which means that such code should not be encouraged without first changing the policy (which would then require figuring out the proper migration path for named arguments, both for backward and forward compatibility)

Reply:
OK, I updated the PR!